config/locales/views/circuitverse/ar.yml

```
@@ -0,0 +1,154 @@
+  warranty_related:
+    main_heading: "10. إخلاء المسؤولية عن الضمان"
+    description1: 'أنت تقر بأنك تستخدم CircuitVerse على مسؤوليتك الخاصة. يتم توفير CircuitVerse "كما هو"، وتخلي كيانات CircuitVerse صراحةً مسؤوليتها عن أي وجميع الضمانات، سواء كانت صريحة أو ضمنية، بما في ذلك على سبيل المثال لا الحصر، الدقة، الموثوقية، الملكية، القابلية للتسويق، عدم الانتهاك، الملاءمة لغرض معين أو أي ضمان أو شرط أو ضمان آخر، سواء كان شفهياً أو كتابياً أو إلكترونياً، بما في ذلك، على سبيل المثال لا الحصر، دقة أو اكتمال أي معلومات مقدمة من CircuitVerse. علاوة على ذلك، تخلي كيانات CircuitVerse مسؤوليتها عن أي ضمانات تتعلق بالمحتوى الذي ينشئه المستخدم وأحداث CircuitVerse. لا تضمن كيانات CircuitVerse ومقدمو خدمات الأطراف الثالثة الوصول غير المنقطع إلى CircuitVerse أو عدم حدوث أي فشل أو خطأ أو سهو أو فقدان للبيانات المنقولة، أو أن الخدمات المقدمة عبر CircuitVerse لن تنقل أي فيروسات.'
+    liability_related:
+        main_heading: "11. حدود المسؤولية"
+        description1: "لن تكون كيانات سيركيت فيرس مسؤولة أمامك أو أمام أي طرف ثالث عن أي أضرار مباشرة أو غير مباشرة أو خاصة أو عرضية أو تأديبية من أي نوع، بغض النظر عن طبيعة المطالبة أو سبب الدعوى، حتى لو تم إبلاغ فريق سيركيت فيرس بإمكانية حدوث مثل هذه الأضرار. دون تقييد ما سبق، لن تكون كيانات سيركيت فيرس مسؤولة عن أي خسارة أو ضرر ينشأ عن المحتوى الذي تم إنشاؤه من قبل المستخدم أو حدث يوم سيركيت فيرس."
+    choice_of_language:
+        main_heading: "14. اختيار اللغة"
+        description1: "إذا قدم لك فريق سيركيت فيرس ترجمة لشروط الاستخدام هذه أو لسياسة الخصوصية أو لأي سياسة أخرى، فإنك توافق على أن هذه الترجمة مقدمة لأغراض إعلامية فقط ولا تعدل النسخة الإنجليزية. في حالة وجود أي تعارض بين الترجمة والنسخة الإنجليزية، فإن النسخة الإنجليزية هي التي ستسود."
+    no_waiver:
+        main_heading: "15. عدم التنازل"
+        description1: "لن يُعتبر أي تنازل عن أي بند من شروط الاستخدام هذه تنازلاً مستمراً عن هذا البند أو أي بند آخر، كما أن فشل فريق سيركيت فيرس في فرض أي حق أو حكم بموجب شروط الاستخدام هذه لن يشكل تنازلاً عن هذا الحق أو الحكم."
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)